owlplug-client/src/main/resources/owlplug.css

```
@@ -667,6 +694,103 @@ ProgressIndicator {
+
+.dashboard-scan-indicator {
+    -fx-background-color: -color-accent-subtle;
+    -fx-background-radius: 6px;
+    -fx-padding: 6 12 6 12;
+}
+
+.dashboard-scan-indicator .ikonli-font-icon {
+    -fx-icon-color: -color-accent-fg;
+}
+
+.dashboard-scan-indicator Label {
```

Stylelint (17.13.0)

[error] Expected "Label" to be "label" (selector-type-case)

(selector-type-case)